docs/sources/developers/http_api/apis.md

```diff
@@ -42,11 +42,27 @@ Groups organize related functionality into logical collections. For example `das
 
 These APIs will also uses semantic versioning with three stability levels:
 
-| Level | Format     | Description                                                                 | Use Case                 |
-| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ |
-| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features |
-| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use |
-| GA    | `v1`       | Generally Available. Stable with backward compatibility guarantees          | For production use       |
+| Level | Format     | Description                                                                 | Use Case                 | Enabled By Default? |
+| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ | ------------------- |
+| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features | No                  |
+| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use | No                  |
+| GA    | `v1`       | Generally Available. Stable with backward compatibility guarantees          | For production use       | Yes                 |
+
+#### Alpha
+
+Alpha versions should not be served unless explicitly enabled by a feature flag, and should be considered completely experimental and subject to major changes.
+An Alpha version may undergo breaking changes without adding an additional version, and should not be relied upon by production workflows.
+
+#### Beta
+
+Beta versions should not contain breaking changes in the schema, but still may be subject to changes in handling logic or semantics.
+Breaking schema changes require a new published beta version (such as publishing `v1beta2` for breaking changes to the `v1beta1` schema).
+While beta versions are no longer considered experimental like alpha versions, they should still be disabled by default and require a feature flag to enable.
```

Comment:
IIUC, we are moving to:
https://kubernetes.io/docs/reference/using-api/#enabling-or-disabling

rather than feature flags for each API.

Unlike k8s, I think we *should* enable beta by default -- but that is maybe a different discussion